scripts/check-monitoring-infrastructure.sh

```
@@ -0,0 +1,244 @@
+    "openshift:training_image_preference:sum"
+    "openshift:training_framework_usage:sum"
+)
+
+for rule in $RULES; do
+    # Check if rule follows openshift: prefix (required for telemetry)
+    if [[ $rule == openshift:* ]]; then
+        echo -e "${GREEN}✓${NC} Recording rule '$rule' follows correct openshift: pattern"
+        ((VALID_RULES++))
+        
+        # Check if it's one of our expected telemetry rules
+        if [[ " ${EXPECTED_RULES[@]} " =~ " ${rule} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)